src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,45 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
```

Comment:
The access modifier for the method `getStatistic` should not be `public` if it is only intended to be used within this class. If this method is indeed used outside, then it can remain public.